CMakeModules/FindclBLAS.cmake

```diff
@@ -1,16 +1,35 @@
-# - Try to find clBLAS
-#  Once done this will define
-#
-#  CLBLAS_FOUND - system has clBLAS
-#  CLBLAS_INCLUDE_DIRS - location of clBLAS.h
-#  CLBLAS_LIBRARIES - location of libclBLAS
-#  
-
-FIND_PACKAGE( PackageHandleStandardArgs )
-
-FIND_PATH(CLBLAS_INCLUDE_DIRS clBLAS.h)
-FIND_LIBRARY(CLBLAS_LIBRARIES clBLAS ENV LD_LIBRARY_PATH)
-
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(clBLAS DEFAULT_MSG CLBLAS_LIBRARIES CLBLAS_INCLUDE_DIRS)
-
-MARK_AS_ADVANCED(CLBLAS_INCLUDE_DIRS CLBLAS_LIBRARIES)
\ No newline at end of file
+# - Try to find clBLAS
+#  Once done this will define
+#
+#  CLBLAS_FOUND - system has clBLAS
+#  CLBLAS_INCLUDE_DIRS - location of clBLAS.h
+#  CLBLAS_LIBRARIES - location of libclBLAS
+#  
+
+IF(CLBLAS_INCLUDE_DIRS)
+  # Already in cache, be silent
+  set (CLBLAS_FIND_QUIETLY TRUE)
+ENDIF (CLBLAS_INCLUDE_DIRS)
+
+FIND_PATH(CLBLAS_ROOT_DIR
+    NAMES include/clBLAS.h
+    HINTS /usr/local/ ${CLBLAS_ROOT} 
+    DOC "clBLAS root directory.")
+    
+FIND_PATH(_CLBLAS_INCLUDE_DIRS
+    NAMES clBLAS.h
+    HINTS ${CLBLAS_ROOT_DIR}/include
+    DOC "clBLAS Include directory")
+
+FIND_LIBRARY(_CLBLAS_LIBRARY
+    NAMES CLBLAS
+    HINTS ${CLBLAS_ROOT_DIR}/lib64/import)

```

Comment:
Hi @abergeron, 
I have added your suggested change, plus another hint with "lib32".
Also, I changed line 16 to have $ENV{CLBLAS_ROOT} instead of ${CLBLAS_ROOT}, because otherwise it still wasn't finding the libraries for me: clearly yesterday I'd set the paths explicitly. I don't know CMake at all, so I'd rather run this change with you before submitting a pull request. Please have a look at the modified FindclBLAS.cmake in my branch https://github.com/thebucc/libgpuarray and let me know if it's ok.